<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Anime Monitor</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Listado</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>648</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_3">
                  <property name="text">
                   <string>Cancelar</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_2">
                  <property name="text">
                   <string>Actualizar</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Descargar todo</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTableWidget" name="listado">
                <property name="horizontalScrollBarPolicy">
                 <enum>Qt::ScrollBarAsNeeded</enum>
                </property>
                <property name="sizeAdjustPolicy">
                 <enum>QAbstractScrollArea::AdjustIgnored</enum>
                </property>
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="verticalScrollMode">
                 <enum>QAbstractItemView::ScrollPerPixel</enum>
                </property>
                <property name="horizontalScrollMode">
                 <enum>QAbstractItemView::ScrollPerPixel</enum>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
                <property name="columnCount">
                 <number>4</number>
                </property>
                <column>
                 <property name="text">
                  <string>Nombre del anime</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Episodio</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Fecha</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Servidor</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Registro</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Busqueda</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QWidget" name="top_panel_search" native="true">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QPushButton" name="pushButton_5">
                     <property name="autoFillBackground">
                      <bool>false</bool>
                     </property>
                     <property name="text">
                      <string>Recargar</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeType">
                      <enum>QSizePolicy::Expanding</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>368</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QComboBox" name="comboBoxProviders">
                     <property name="minimumSize">
                      <size>
                       <width>180</width>
                       <height>0</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="Line" name="line">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>23</height>
                      </size>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="searchTextBox">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>220</width>
                       <height>0</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="searchButton">
                     <property name="text">
                      <string>Buscar</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QSplitter" name="splitter">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="opaqueResize">
                   <bool>true</bool>
                  </property>
                  <property name="handleWidth">
                   <number>4</number>
                  </property>
                  <property name="childrenCollapsible">
                   <bool>false</bool>
                  </property>
                  <widget class="QListWidget" name="listAnimes">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>250</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="baseSize">
                    <size>
                     <width>250</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="alternatingRowColors">
                    <bool>true</bool>
                   </property>
                  </widget>
                  <widget class="QWidget" name="widget" native="true">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_10">
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_9">
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_5">
                        <item>
                         <widget class="QGraphicsView" name="graphicsView">
                          <property name="maximumSize">
                           <size>
                            <width>156</width>
                            <height>16777215</height>
                           </size>
                          </property>
                          <property name="styleSheet">
                           <string notr="true">QGraphicsView {
	background-color: #F0F0F0;
	border-radius: 5px;
}</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <layout class="QVBoxLayout" name="verticalLayout_8">
                          <item>
                           <widget class="QTextEdit" name="textEdit">
                            <property name="maximumSize">
                             <size>
                              <width>16777215</width>
                              <height>160</height>
                             </size>
                            </property>
                            <property name="styleSheet">
                             <string notr="true">QTextEdit {
	background-color: #F0F0F0;
	border-radius: 5px;
}</string>
                            </property>
                            <property name="readOnly">
                             <bool>true</bool>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <spacer name="verticalSpacer_2">
                            <property name="orientation">
                             <enum>Qt::Vertical</enum>
                            </property>
                            <property name="sizeType">
                             <enum>QSizePolicy::Expanding</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                             <size>
                              <width>20</width>
                              <height>40</height>
                             </size>
                            </property>
                           </spacer>
                          </item>
                         </layout>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <spacer name="verticalSpacer_3">
                        <property name="orientation">
                         <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>20</width>
                          <height>40</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>1</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="notification">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBarProviders">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>ListResultClicked()</slot>
 </slots>
</ui>
